example/index.html

```diff
@@ -103,6 +103,9 @@
   });
 
   window.conn = a;
+  window['Connector'] = Connector
+  window['Model'] = Model
+  window['RelationTypes'] = RelationTypes
 
```

Comment:
i missed out the pattern i shouldn't use the brackets 
❌window['Model']  = ...
✔️window.Model = ...